### PR TITLE
Remove dead code and add cover `home.py` test

### PR DESCRIPTION
**Pull request by bc1cindy**
### What is this PR for?

Remove unused passphrase and customize prompts that no longer appear in device interface. Update tests to reflect actual device behavior without non-existent prompts. Add comprehensive test coverage for all home page methods including edge cases and error paths.

### Changes made to:
- [ ] Code
- [x] Tests
- [ ] Docs
- [ ] CHANGELOG

### Did you build the code and tested on device?
- [ ] Yes

### What is the purpose of this pull request?
- [ ] Bug fix
- [ ] New feature
- [ ] Docs update
- [ ] Other

### CODE REVIEW

**Comment by bc1cindy** [comment]
> > @odudex I tested it on the simulator and a real device, but they didn't show up for me, so I removed them. I'll consider them as if they were there. Thanks!
> 
> Just to ensure we are on the same page, did you run your tests AFTER (not during) loading the mnemonic? Going to Wallet -> Passphrase or Wallet->Customize?

Ahh, I was doing it during, now I saw the messages in the right place, thanks for clarifying @odudex !! 

The tests are now aligned.

File: tests/pages/home_pages/test_home.py [hunk 1]
```diff
@@ -1583,3 +1606,141 @@ def test_sign_spent_and_self(mocker, m5stickv, tdata):
...
+    ctx = create_ctx(mocker, BTN_SEQUENCE, wallet=wallet)
+
+    home = Home(ctx)
+    result = home.sign()
+
+    assert result == MENU_CONTINUE
+    assert ctx.input.wait_for_button.call_count == len(BTN_SEQUENCE)
+
+
+def test_sign_message_status_return(mocker, amigo, tdata):
```
Comment (odudex) [inline tests/pages/home_pages/test_home.py:1703]: The goal of this test is not clear. It mocks a result, then check the mock itself.

File: tests/pages/home_pages/test_home.py [hunk 2]
```diff
@@ -1583,3 +1606,141 @@ def test_sign_spent_and_self(mocker, m5stickv, tdata):
...
+
+    home = Home(ctx)
+    mocker.patch.object(home, "sign_message", return_value="SUCCESS")
+    result = home.sign()
+
+    assert result == "SUCCESS"
+    assert ctx.input.wait_for_button.call_count == 2
+
+
+def test_load_psbt_invalid_method(mocker, m5stickv, tdata):
```
Comment (odudex) [inline tests/pages/home_pages/test_home.py:1726]: please elaborate on "invalid_method" and the test objective.
